packages/sdk/typescript/human-protocol-sdk/src/utils.ts

```diff
@@ -45,13 +46,16 @@ export const throwError = (e: any) => {
  * @param {string} url
  * @returns
  */
-export const isValidUrl = (url: string) => {
-  try {
-    new URL(url);
-    return true;
-  } catch (err) {
-    return false;
-  }
+export const isValidUrl = (url: string): boolean => {
+  if (typeof url !== 'string' || url.trim() === '') return false;
+
+  const localhostRegex = /^https?:\/\/localhost(?::\d{1,5})?(?:\/|$)/i;
+  if (localhostRegex.test(url)) return true;
```

Comment:
Let's remove this and use `require_tld: false` settings instead: when we run services in docker we use service names in URLs (e.g. `http://minio:9000/bucket_name`. `http://reputation-oracle:5000/health/ping), so if we don't accept these - it will be broken